CONTRIBUTING.md

```diff
@@ -418,22 +420,23 @@ For example, the image referenced above actually lives in `superset-frontend/ima
 
 #### OS Dependencies
 
-Make sure your machine meets the [OS dependencies](https://superset.apache.org/docs/installation/installing-superset-from-scratch#os-dependencies) before following these steps.
+Make sure your machine meets the [OS dependencies](https://superset.apache.org/docs/installation/installing-superset-from-scratch#os-dependencies) before following these steps.  
+You also need to install MySQL or [MariaDB](https://mariadb.com/downloads).
 
-Ensure Python versions >3.7, Then proceed with:
+Ensure that you are using Python version 3.7 or 3.8, then proceed with:
 
 ```bash
 # Create a virtual environment and activate it (recommended)
 python3 -m venv venv # setup a python3 virtualenv
 source venv/bin/activate
 
 # Install external dependencies
-pip install -r requirements/local.txt
+pip install -r requirements/testing.txt
```

Comment:
do we need both the `testing` and `local` deps? Seems like we might need both

Reply:
Testing is a superset(😄) of local